src/celestia/celx_celestia.cpp

```diff
@@ -2475,6 +2475,18 @@ static int celestia_getrootcategories(lua_State *l)
     return celx.pushIterable<UserCategory*>(set);
 }
 
+static int celestia_bindtextdomain(lua_State *l)
+{
+    CelxLua celx(l);
+    
+    const char *domain = celx.safeGetString(2, AllErrors, "First argument of celestia:bindtranslationdomain must be domain name string.");
```

Comment:
Please check domainname: "domainname must be a non-empty string."

and plz rename your commit to some concrete "add celestia:bindtextdomain method" or similar, as you don't have support for ssc & friends.